Automattic/wp-calypso
# Verbum Block Editor: Fix pasting and wp.com embed variation

This brings over the Verbum fix here: 91-gh-Automattic/verbum and some of 102-gh-Automattic/verbum.

## Proposed Changes

This check if the pasted text is a URL at all before checking if it's an embed URL.

## Testing Instructions

1. In `packages/verbum-block-editor` run `yarn dev --sync`.
2. Sandbox widgets.wp.com
3. Copy complicated HTML with links inside it.
4. Paste in the comment field. It should be parsed successfully.
5. Paste a wp.com blogpost article from a *.wordpress.com domain, it should be embedded.
